test/Interop/Cxx/operators/member-inline.swift

```diff
@@ -18,6 +18,12 @@ OperatorsTestSuite.test("LoadableIntWrapper.plus (inline)") {
 }
 #endif
 
+OperatorsTestSuite.test("LoadableIntWrapper.tilde (inline)") {
+  let wrapper = LoadableIntWrapper(value: 42)
+  let not42 = ~wrapper
+  expectEqual(~42, not42)
```

Comment:
Nit: can you make the first argument here a constant (not something computed). So in this case `-43`. This is a good general rule of thumb for tests because it makes it easier to catch bugs.